### PR TITLE
Update modularchangling.dm [sic]

### PR DESCRIPTION
**Pull request by ihadtoregisterforthis**
C'mon now

### CODE REVIEW

**Comment by ghost** [comment]
Kek

**Review by PJB3005 — APPROVED** [review]
Jesus christ.

File: code/game/gamemodes/changeling/modularchangling.dm [hunk 1]
```diff
@@ -185,9 +185,9 @@ var/list/datum/power/changeling/powerinstances = list()
 	verbpath = /mob/proc/changeling_engorgedglands
 
 /datum/power/changeling/DigitalCamoflague
```
Comment (ihadtoregisterforthis) [inline code/game/gamemodes/changeling/modularchangling.dm:187]: Left this just in case

Comment (Exxion) [reply code/game/gamemodes/changeling/modularchangling.dm:187]: It was misspelled in two different ways. Excellent

**Pull request merged**